src/cpp/client/ProxyClient.cpp

```diff
@@ -602,5 +604,18 @@ bool ProxyClient::delete_object_unlock(
     return rv;
 }
 
+ProxyClient::State ProxyClient::get_state()
+{
+    using namespace std::chrono;
+    return (duration_cast<milliseconds>(steady_clock::now() - timestamp_) < CLIENT_DEAD_TIME)
+            ? State::alive
+            : State::dead;
```

Comment:
```C++
return state_ = (duration_cast<milliseconds>(steady_clock::now() - timestamp_) < CLIENT_DEAD_TIME)
            ? State::alive
            : State::dead;
```

And change the `get_` prefix?

Reply:
Do you prefer `state()` instead of `get_state()`?
I modified it.